runelite-client/src/main/java/net/runelite/client/ui/overlay/infobox/Timer.java

```diff
@@ -44,9 +46,9 @@ public Timer(long period, ChronoUnit unit, BufferedImage image, Plugin plugin)
 
 		Preconditions.checkArgument(period > 0, "negative period!");
 
-		startTime = Instant.now();
-		duration = Duration.of(period, unit);
-		endTime = startTime.plus(duration);
+		this.startTime = Instant.now();
```

Comment:
These `this.` additions can be removed? I don't see any local variables in this scope that would hide the class fields.

Reply:
Tbh I like to prepend `this.` because it makes it clearer that we're not writing to local variables. But I don't care enough so I'll just change it